compiler/damlc/tests/daml-test-files/ChoiceSyntaxes.daml

```diff
@@ -0,0 +1,32 @@
+-- Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates.
+-- All rights reserved.
+
+-- Check that all syntaxes for defining choices can be parsed.
+daml 1.2
+module ChoiceSyntaxes where
+
+template T with
+    p : Party
+  where
+    signatory p
+
+    choice C1 : () with
+        x : ()
+      controller p
```

Comment:
Maybe change this to something that actually makes use of the fact that the choice argument can be used in the controller? I’m sure we have other tests for that but it doesn’t do any harm to use it here as well and makes it easier to quickly see what’s different about the 3 cases.

Reply:
Good idea. Done.